impl/src/test/java/org/glassfish/json/tests/JsonReaderDuplicateKeyTest.java

```diff
@@ -0,0 +1,40 @@
+package org.glassfish.json.tests;
```

Comment:
missing license

Reply:
Added license / copyright header to this file.